### DISCUSSION TITLE
how to fix this type of error ?

### DISCUSSION THREAD

**Discussion by rahman-cmd** [community/community#112261] [Programming Help]
### Body

Error: The method 'hasProperty' isn't defined for the class 'EventTarget'.
 - 'EventTarget' is from 'package:web/src/dom/dom.dart'
 ('/C:/Users/abrahman/AppData/Local/Pub/Cache/hosted/pub.dev/web-0.3.0/lib/src/dom/dom.dart').
  } else if (e.hasProperty('msHidden'.toJS).toDart) {

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by kuciapakrystian** [comment]
It seems like you're encountering an error in Dart code related to the EventTarget class and the hasProperty method. The error indicates that the hasProperty method is not defined for the EventTarget class.

The EventTarget class in Dart is typically used for objects that can receive events, such as HTML elements. However, the hasProperty method doesn't seem to be a standard method in the Dart EventTarget class.

If you're trying to check whether an event has a specific property, you might need to use a different approach. Here's how you might typically check for the existence of a property in a JavaScript event object:
if ('msHidden' in e) {
    // Property exists
} else {
    // Property doesn't exist
}

In Dart, you could use the [] operator to access properties dynamically:
if (e['msHidden'] != null) {
    // Property exists
} else {
    // Property doesn't exist
}
Replace e['msHidden'] with the appropriate property access based on your event object's structure.

**Comment by calenzo** [comment]
Do you can show the code ?

Do if in your case is an object, try: 
`!!myObject?.msHidden`
